### ISSUE TITLE
Add more columns (Shipping Phone and Billing Phone) to WooCommerce/Customers

### ISSUE THREAD

**Issue by datgausaigon** [woothemes/woocommerce#30505]
Hi,
In WooCommerce/Customers (admin)
We have default columns is:

- Name
- Username	
- Last Active
- Date Registered
- Email	
- Orders
- Total Spend
- AOV (Average Order Value)
- Country / Region
- City
- Region
- Postal Code

In next release (5.6) will have a update:
Enhancement – Added support for Shipping Phone, in addition to Billing Phone in email templates and in the Order’s edit screen. #30097

Phone Number is a important field help we manage Customer.

Can I request a new enhancement is: Add new columns: Shipping Phone and Billing Phone to WooCommerce/Customers?

Thanks

**Comment by tammullen** [comment]
Hi @datverse 

Thank you for taking the time to share this idea, we really appreciate your help. 

While this is clearly a great improvement/idea we won’t be able to tackle it in the upcoming weeks. We’re going to add it to our backlog where it will be considered for future releases.

Stay tuned for updates.

**Comment by datgausaigon** [comment]
Thank you for your answer. I hope that my small suggestions help the woo community.
Thanks

**Comment by datgausaigon** [comment]
Hi @tammullen,
Can this new feature be updated to the upcoming 5.8 or 5.9 version? Thanks

**Comment by redifmail** [comment]
Waiting for this feature.

**Comment by datgausaigon** [comment]
@redifmail Yup, in release note of upcoming 6.1 beta still don't have this feature. 
Hope WooCommerce team make this soon.

**Comment by ELY3M** [comment]
I need this too.

**Comment by datgausaigon** [comment]
Hi @tammullen @ObliviousHarmony how about this issue?

**Comment by xue28** [comment]
6080697-zen

**Comment by Markmcl** [comment]
Is there any planned progress on this please?

**Comment by KoolPal** [comment]
@tammullen Any update on this being made available? It has been over 2 years now. Thanks

**Comment by bludwig** [comment]
Anything to report on this potential addition?

**Comment by Mehran91z** [comment]
+1 for this.
It has not been updated from `Aug 16, 2021`! This feature should not be that so hard to develop!

**Comment by KoolPal** [comment]
@vedanshujain  Would you please take a look at this and open an enhancement request?

Cursory search in wordpress.org forum shows DEMAND for this FEATURE!
https://wordpress.org/support/topic/customizing-customer-columns-in-admin/
https://wordpress.org/support/topic/add-phone-to-customer-table-list/
https://wordpress.org/support/topic/add-more-column-to-woocommerce-customers/

Thanks

**Comment by lailagmelo** [comment]
User request for this feature: 9324608-zd-a8c

**Comment by rubnrestore** [comment]
It's bonkers that phone numbers are not included in customer exports and all the more so that this enhancement request has been backlogged for four years! Please make this enhancement!

**Comment by sanyokdb** [comment]
Please tell me if it is possible to insert a phone number into the list of clients? I could not find it myself. Thank you

**Comment by szymonmil** [comment]
What is happening with this request?

**Comment by jamesckemp** [comment]
Hi @datgausaigon - thanks for the report.

I know this has been open for a very long time, and I appreciate your patience. We're currently going through some older WooCommerce issues, identifying what still needs fixing, and tidying up what's already fixed or out of scope.

I've taken a look and marked this ready for the engineering team to pick up.

It makes sense to me to enable this data to be shown, especially considering the numerous requests for it!
